weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -44,6 +45,9 @@ export const ExportSelector = ({
   rightmostButton?: boolean;
 }) => {
   const [selectionState, setSelectionState] = useState<SelectionState>('all');
+  const [downloadLoading, setDownloadLoading] = useState<ContentType | null>(
```

Comment:
Likely people aren't downloading more than one content type at a time, but maybe we should be disabling the ClickableOutlinedCardWithIcon if we want to only allow one at a time?

Reply:
added a color change, cursor change, disabled clicking other cells, and prevented double clicking